ui-core/src/participant/dashboard/Alert.tsx

```diff
@@ -42,7 +79,7 @@ export const Alert = (props: AlertProps) => {
       {renderedIcon && <FontAwesomeIcon className="me-3 fa-xl" icon={renderedIcon} />}
       <div className="flex-grow-1">
```

Comment:
I think we should upgrade both the title and text to support markdown -- I wouldn't be surprised if a lot of the messages will contain links